### PR TITLE
Adding logging and session data subsection.

### PR DESCRIPTION
**Pull request by Oglopf**
**Modify this link to include the branch name, and possibly the page this PR modifies**:

https://osc.github.io/ood-documentation-test/add-faq-logs/

**Add your description here**
Adding section for logging and session data information.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201735133575781/1201908247003701) by [Unito](https://www.unito.io)

### CODE REVIEW

File: source/monitoring/logging.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+.. warning::
+
+    There are no entries for failed logins.
+
+**NGINX Logs**
+
+For logs relating to issues about:
+
+ - Users trying to connect to a :ref:`login-node <glossary>` from their PUN check:
+
```
Comment (johrstrom) [inline source/monitoring/logging.rst]: I would say first that the nginx logs are the output of the PUN.

I don't think that related issue is right (you're referencing the shell app? it doesn't log much).

Related issues are things specific to users (not site wide).  I can only think of these two.
- Debugging issues related to job submission. Specifically checking commands being issued to the scheduler (sbatch, qsub, etc) by searching for `execve`.
- Issues related to PUNs crashing and/or pages not rendering correctly.

Comment (johrstrom) [reply source/monitoring/logging.rst]: edited the comment above.

File: source/monitoring/logging.rst [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+In general the session data or job submission files for apps across the dashboard, such as the Job Composer, Batch Connect, 
+or Frame-renderer, all start from the root of:
+
+:file:`~/ondemand/data/sys/`
+
+**Batch Connect**
+User session data for batch connect apps can be seen from a *user's* home directory at:
+
+:file:`~/ondemand/data/sys/dashboard/batch_connect/db/<session-file>`
```
Comment (johrstrom) [inline source/monitoring/logging.rst]:
```suggestion
:file:`~/ondemand/data/sys/dashboard/batch_connect/sys/<app>/output/<session id>`
```

Are these the files you're referencing?  I generally look at the output of the job, not the db files.

File: source/monitoring/logging.rst [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+.. note::
+
+    These logs by default have ``root:root`` ownership. It may be beneficial 
+    to ``chown`` an appropriate staff group on these files for troubleshooting down the road.
+
+Session Data
+------------
+Though not logs per-se, these files will provide information around connections between the user's PUN and a 
+:ref:`login-node<glossary>`, such as an interactive app like Jupyter.
```
Comment (johrstrom) [inline source/monitoring/logging.rst]: So jobs run on compute nodes, not login nodes so I'd just remove that bit.

They also _are_ logs becuase that's where we tell the scheduler to put the stdout and stderr and apps typically log other info there.

Lastly - they're `information around connections` so much as they are `the working directory of the job as well as the location of stdout and stderr output files (typically just output.log)`.

File: source/monitoring/logging.rst (hunk 2 above)
Comment (johrstrom) [reply source/monitoring/logging.rst]: Reading the rest of it you are referencing the db. I'd drop that and replace it with what's the `staged_root` of the job that has all the template files, output logs and so on.

I rarely look at a user's DB file - unless there's something really funny going on.

File: source/monitoring/logging.rst (hunk 1 above)
Comment (Oglopf) [reply source/monitoring/logging.rst]: I reworded the section to try and capture this as best I could. Let me know what you think.

File: source/monitoring/logging.rst (hunk 3 above)
Comment (Oglopf) [reply source/monitoring/logging.rst]: Updated to reflect the correct node type and also tried to capture the connections and location information as well. Let me know what you think.

File: source/monitoring/logging.rst [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+Suppose a user is having trouble connecting to a Codeserver session they created. 
+To see what data is being used by this batch connect app for the connection, look in:
+
+.. code-block:: sh
+    
+    cat ~/ondemand/data/sys/dashboard/batch_connect/sys/<app>/output/<session id>
+    
+This will output a json object which will give the information to debug 
+any issues this session is having when connecting, such as ``id``, ``token``, ``cluster_id``, 
+and ``job_id``.
```
Comment (johrstrom) [inline source/monitoring/logging.rst]: I think the example here is still around looking at the db file?  I think `output.log` here is pretty reliable. Like every BC app is going to have the scheduler redirect stdout and stderror to the `<%= staged_root %>/output.log` unless they override it which I'd say is a rare edge case - or at least a change made by an informed user (so not the reader of this document).

Comment (Oglopf) [reply source/monitoring/logging.rst]: Yes thank you for catching that. I updated the example and tried to include some useful information as to what that `output.log` would capture which the admin may be interested in for troubleshooting as well.

**Review by johrstrom — APPROVED** [review]
LGTM

**Pull request merged**